test/app/index.html

```diff
@@ -11,9 +11,8 @@
         paths: {
           // We have to define a path to Scribe otherwise it will load the file
           // but will not pass along the 'scribe' AMD module.
-          'scribe-common': '../../bower_components/scribe-common',
           'lodash-amd': '../../bower_components/lodash-amd',
-          'immutable': '../../bower_components/immutable'
+          'immutable': '../../bower_components/immutable/dist/immutable'
```

Comment:
I'll try a fresh checkout, Scribe should pull in the dependency as it needs it rather than the plugin.